hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieSanity.java

```diff
@@ -37,6 +37,7 @@
 /**
  * Smoke tests to run as part of verification.
  */
+@Disabled("HUDI-8274")
```

Comment:
I would say let's remove it if such tests are already covered by other functinal tests.

Reply:
It's covered in HUDI-8274.  I think there is still value to rewrite the integration tests in a different way, so I keep them here for reference.